openapi/components/schemas/Applications/ApplicationInstanceConfiguration.yaml

```
@@ -0,0 +1,37 @@
+type: object
+required:
+  - settings
+description: |-
+  Application instance configuration.
+
+  Only user can change settings when `configuredBy` is `user`.
+
+  Only the application itself can change settings when `configuredBy` is `application`.
+properties:
+  settings:
+    description: Settings that are available for the user to complete when they install the application.
+    type: object
+    additionalProperties:
+      type: string
```

suggestion: probably good to describe this field also (I'm not sure if my suggestion is correct)
```suggestion
    additionalProperties:
      type: string
      description: Additional properties that the user may want to add when installing an application.
```

---

Not quite correct. The structure of this object depends on what kind of schema is described in properties object and may vary. It is not defined by user.